WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingNavigationController.swift

```diff
@@ -24,14 +24,32 @@ class PrepublishingNavigationController: LightNavigationController {
                 return Constants.iPadPreferredContentSize
             }
 
-            guard  let visibleViewController = viewControllers.last else {
+            guard let visibleViewController = viewControllers.last else {
                 return .zero
             }
 
             return visibleViewController.preferredContentSize
         }
     }
 
+    override func pushViewController(_ viewController: UIViewController, animated: Bool) {
+        super.pushViewController(viewController, animated: animated)
+
+        if let bottomSheet = self.parent as? BottomSheetViewController, let presentedVC = bottomSheet.presentedVC {
```

Comment:
How about moving this to a private function:
```
private func transition() {
        if let bottomSheet = self.parent as? BottomSheetViewController, let presentedVC = bottomSheet.presentedVC {
            presentedVC.transition(to: .collapsed)
        }
    }
```
And avoiding the duplication?

Reply:
Good plan! 👍

Reply:
Done in: https://github.com/wordpress-mobile/WordPress-iOS/pull/15551/commits/deeabfd699c0e53c9c19989f2cf481fc666a5aa4